### PR TITLE
Remove some deprecated columns

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: commons/c2cgeoportal_commons/alembic/main/c75124553bf3_remove_deprecated_columns.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# revision identifiers, used by Alembic.
+revision = 'c75124553bf3'
+down_revision = 'dba87f2647f9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    schema = config['schema']
+    staticschema = config['schema_static']
```
Comment (fredj) [inline commons/c2cgeoportal_commons/alembic/main/c75124553bf3_remove_deprecated_columns.py]: variable is not used

File: commons/c2cgeoportal_commons/alembic/main/c75124553bf3_remove_deprecated_columns.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    op.drop_column('layergroup', 'is_internal_wms', schema=schema)
+    op.drop_column('layergroup', 'is_base_layer', schema=schema)
+
+
+def downgrade():
+    schema = config['schema']
+    staticschema = config['schema_static']
```
Comment (fredj) [inline commons/c2cgeoportal_commons/alembic/main/c75124553bf3_remove_deprecated_columns.py]: variable is not used

File: commons/c2cgeoportal_commons/models/main.py [hunk 3]
```diff
@@ -414,17 +414,6 @@ class LayerGroup(TreeGroup):
             'title': _('Expanded'),
             'column': 2
         }})  # shouldn't be used in V3
```
Comment (fredj) [inline commons/c2cgeoportal_commons/models/main.py]: should the comment be removed ?

**Comment by fredj** [comment]
there's maybe more code that can removed (`git grep is_internal_wms`)

File: commons/c2cgeoportal_commons/models/main.py (hunk 3 above)
Comment (sbrunner) [reply commons/c2cgeoportal_commons/models/main.py]: No because it's steel be the case, but we can't remove it because the script themev1tov2 use it :-)

**Comment by sbrunner** [comment]
Yes, for that we need the multi role to avoid conflicts ...

**Pull request merged**